code/modules/cargo/bounties/medical.dm

```diff
@@ -53,7 +53,7 @@
 /datum/bounty/item/medical/eye
 	name = "Organic Eyes"
 	description = "Station 5's Research Director Willem is requesting a few pairs of non-robotic eyes. Don't ask questions, just ship them."
-	reward = CARGO_CRATE_VALUE * 20
+	reward = CARGO_CRATE_VALUE * 10
```

Comment:
I'd advise keeping the value for the bounties that you can't make by using the prolathe the same like the organic eyes and tongues.

Reply:
This is still an issue because people are capable (and have) dropped half a dozen monkey corpses onto a bounty pad, butchered all of them, and shat out twenty grand in bounties in minutes with access from botany or the public garden

Reply:
If it's the difference between 400 and 800 credits in the current system, then I'd probably agree this needs to get tweaked downward. If it's easy to process more than this is a slightly better place for it.

Reply:
> This is still an issue because people are capable (and have) dropped half a dozen monkey corpses onto a bounty pad, butchered all of them, and shat out twenty grand in bounties in minutes with access from botany or the public garden

I didn't think you could get tongues from butchering monkeys en masse, but I could be wrong about that.